co-log-core/src/Colog/Core/Action.hs

```diff
@@ -270,30 +281,62 @@ cmapM :: Monad m => (a -> m b) -> LogAction m b -> LogAction m a
 cmapM f (LogAction action) = LogAction (f >=> action)
 {-# INLINE cmapM #-}
 
--- | @divide@ combinator from @Divisible@ type class.
+{- | @divide@ combinator from @Divisible@ type class.
+
+>>> logString = LogAction putStrLn
+>>> logInt = LogAction print
+>>> "ABC" &> divide (\s -> (s, length s)) logString logInt
+ABC
+3
+-}
 divide :: (Applicative m) => (a -> (b, c)) -> LogAction m b -> LogAction m c -> LogAction m a
 divide f (LogAction actionB) (LogAction actionC) = LogAction $ \(f -> (b, c)) ->
     actionB b *> actionC c
 
--- | @conquer@ combinator from @Divisible@ type class.
+{- | @conquer@ combinator from @Divisible@ type class.
+
+Concretely, this is a logAction that does nothing:
```

Comment:
Instead of `logAction` it's better to write `'LogAction'` to convert it into link